tools/third_party.py

```diff
@@ -64,6 +64,8 @@ def fix_symlinks():
     remove_and_symlink("third_party/v8/build_overrides",
                        root("build_overrides"), True)
     remove_and_symlink("third_party/v8/testing", root("testing"), True)
+    remove_and_symlink("../third_party/v8/tools/clang",
+                       root("tools/clang"), True)
```

Comment:
necessary?

Reply:
Yes - the link is used somewhere (forgot where).
Try to remove this  and delete /tools/clang to find out where :)